cob_omni_drive_controller/CMakeLists.txt

```diff
@@ -6,27 +6,19 @@ find_package(catkin REQUIRED COMPONENTS angles controller_interface geometry_msg
 find_package(Boost REQUIRED)
 
 catkin_package(
+  CATKIN_DEPENDS angles controller_interface geometry_msgs hardware_interface nav_msgs sensor_msgs std_srvs tf urdf
```

Comment:
for the sake of readability: shouldn't we have all those on seperate, correctly indented, lines?